types/stylelint/index.d.ts

```diff
@@ -858,11 +858,24 @@ declare namespace stylelint {
 		quiet?: boolean;
 		quietDeprecationWarnings?: boolean;
 		validate?: boolean;
+		suppressAll?: boolean;
+		suppressLocation?: string;
+		suppressPrune?: boolean;
+		suppressRule?: string[];
 	};
 
 	/** @internal */
 	export type FixMode = 'lax' | 'strict';
 
+	/** @internal */
```

Comment:
```
	/** @experimental */
```

see https://github.com/stylelint/stylelint/pull/8564/files#diff-3c13621383c73beb724ce15f4f48ae834d45b119794212ea8c9b6765f94536cb

Reply:
Fixed it.
[Refactor to unify emitWarning function](https://github.com/stylelint/stylelint/pull/8564/commits/0b406eb23ce8c9f6186f842b1733154c82a81028)